aws_lambda_builders/workflows/nodejs_npm/actions.py

```diff
@@ -81,11 +81,10 @@ class NodejsNpmInstallAction(BaseAction):
     DESCRIPTION = "Installing dependencies from NPM"
     PURPOSE = Purpose.RESOLVE_DEPENDENCIES
 
-    def __init__(self, artifacts_dir, subprocess_npm):
+    def __init__(self, install_dir, subprocess_npm):
```

Comment:
renamed to generalize (e.g. now we can install in the source directory as well, so this should not be specific to the artifacts dir)

Reply:
Something to consider when changing the arguments, let's make a callout to not changing the RPC protocol version because we did not touch the workflow args. https://github.com/aws/aws-lambda-builders/blob/develop/aws_lambda_builders/__init__.py#L8

If not considered, we should think of adding that into PR template so that we ask ourselves that question.

Reply:
actually, _when_ should we change the RPC protocol version? Is it when we change any of the params passed in the request [here](https://github.com/aws/aws-lambda-builders/blob/develop/aws_lambda_builders/__main__.py#L88)? Or is it only on breaking changes for the params there?